tests/tizen_capi/unittest_tizen_capi.cc

```diff
@@ -1453,11 +1453,9 @@ TEST (nnstreamer_capi_util, availability_fail_04_n)
 
   status = ml_check_nnfw_availability (ML_NNFW_TYPE_NNFW, ML_NNFW_HW_NPU_SR, &result);
   EXPECT_EQ (status, ML_ERROR_NONE);
-  EXPECT_EQ (result, false);
```

Comment:
Is this related to the subject (adding test cases for 'auto' of tensor_filter)?

Reply:
Thank you. Not relate with this PR. I will speperate the PR.